src/frontend/utils/test/validators.test.ts

```diff
@@ -1,9 +1,19 @@
-import { FeltState, Valideringsstatus } from '@navikt/familie-skjema';
+import { act, renderHook } from '@testing-library/react-hooks';
```

Comment:
du er så flink til å introdusere verktøy og teknikker i prosjektene 💯  fint og lett for andre å ta i bruk og lære da

Reply:
Tusen takk 🤗  Dette var jo riktignok "stjålet" fra ba søknad da 💸